### PR TITLE
gnomeExtensions.gsconnect: 43 -> 44

### PR DESCRIPTION
**Pull request by doronbehar**
###### Motivation for this change

Gnome 3.38 - https://github.com/NixOS/nixpkgs/issues/95248 .

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Review by jtojnar — APPROVED** [review]
Changes look okay https://github.com/andyholmes/gnome-shell-extension-gsconnect/releases/tag/v44

**Comment by jtojnar** [comment]
Looks like installed tests still fail but exactly as before.

**Comment by doronbehar** [comment]
Will merge once I see this working after I will use my system with this merged to my nixpkgs fork.

**Comment by doronbehar** [comment]
Works.

**Pull request merged**
